api/models.py

```diff
@@ -0,0 +1,111 @@
+import enum
+from app import db, ma
+from datetime import datetime, date
+from sqlalchemy import ForeignKey, func, text, Date
```

Comment:
ik you got all of this written out but have you seen sqlc? https://docs.sqlc.dev/en/latest/tutorials/getting-started-postgresql.html

Reply:
if changing the backend from python to go is considered a good move at this point then i can look into learning go and setting up a go backend

Reply:
you've made it this far with python!! don't sweat it